app/components/elements/forms/input_collection_checkboxes_component.html.erb

```diff
@@ -2,8 +2,8 @@
   <%= tag.div class: div_classes do %>
 
     <% disabled = disabled_values.include?(builder.object) %>
-    <%= builder.check_box(class: 'form-check-input', data: input_data, disabled:, readonly: disabled) %>
-    <%= builder.label(class: 'form-check-label') %>
+    <%= builder.check_box(class: disabled ? disabled_checkbox_classes : checkbox_classes, data: input_data, disabled:, readonly: disabled) %>
+    <%= builder.label(class: disabled ? disabled_label_classes : label_classes) %>
```

Comment:
What would you think about moving this logic to the Ruby class within the component? I.e., having the Ruby class check `disabled` and set the various classes accordingly?